server.js

```diff
@@ -72,10 +77,19 @@ console.log('config', config);
 
 // //////////////////////////////////////////////////////////////////////
 
-var smtp = nodemailer.createTransport('SMTP', config.mailTransportConfig);
+var smtp = nodemailer.createTransport({
+  host: process.env.MAILGUN_SMTP_SERVER,
+  port: process.env.MAILGUN_SMTP_PORT,
+  secure: false,
+  auth: {
+    user: process.env.MAILGUN_SMTP_LOGIN,
+    pass: process.env.MAILGUN_SMTP_PASSWORD
+  }
+});
```

Comment:
Can you make the mailer configuration conditional on the presence of `config.mailTransportConfig` and the `MAILGUN.*` configuration variables?  Something like
```
var smtp;
if (config.mailTransportConfig) {
  smtp = nodemailer.createTransport('SMTP', config.mailTransportConfig);
} else if (env.MAILGUN_SMTP_SERVER) {
  smtp = nodemailer.createTransport({
    host: process.env.MAILGUN_SMTP_SERVER,
    port: process.env.MAILGUN_SMTP_PORT,
    secure: false,
    auth: {
      user: process.env.MAILGUN_SMTP_LOGIN,
      pass: process.env.MAILGUN_SMTP_PASSWORD
    }
  });
} else {
  console.log('email sending not configured');
}
```